extensions/mqtt/processors/AbstractMQTTProcessor.h

```diff
@@ -157,7 +157,6 @@ class AbstractMQTTProcessor : public core::Processor {
   std::string securityPrivateKeyPassWord_;
 };
 
-REGISTER_RESOURCE(AbstractMQTTProcessor);
```

Comment:
The appropriate thing would be to remove this class, actually. I'll try to put the details in a ticket, but the priority is low since it's an extension